Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM php:alpine3.10
+
+WORKDIR /workdir
+
+RUN apk add composer
+
+COPY composer.json composer.json
+RUN composer install
```

Comment:
pretty sure we don't need `composer` to run the application, can we have a multistage build?

Reply:
Yes, working on it :)

Reply:
Done 👌